### ISSUE TITLE
Someone is plagiarizing your work

### ISSUE THREAD

**Issue by thagberg** [adisve/chess-game#3]
Wasn't sure how else to notify you since I couldn't find your email, but someone is plagiarizing your work on this game (listed it on their own portfolio website and posted your code on their own github but failed to remove your name from the comments).

https://github.com/partheshp/chess-game/blob/main/Game/GameState/GameState.h#L2

https://partheshp.github.io/PartheshParikh.github.io/

**Comment by defaultdino** [comment]
@thagberg 

Thank you! I have let @partheshp know and hopefully they remove the code and at least fork my existing repo.

**Comment by partheshp** [comment]
@adisve  I have removed the project from everywhere. I am extremely sorry as this came across in a bad light from my side. Actually, I came across the project while I was learning C++, and downloaded the code for my reference. I have a habit of forking projects, as can be seen from my repo (e.g., Sign Language Converter), and even when I use someone's work, I don't remove their credits. The issue with this project was that after I downloaded the code, I forgot to fork it!
Also, @thagberg, thank you for pointing this out. I will make sure such things don't happen in the future.
I will write a new version of the game myself and upload it.
